jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -32,18 +32,30 @@
   private final byte flags;
   private final Map<String, String> baggage;
   private final String debugId;
+  private final TracingFactory tracingFactory;
 
   public JaegerSpanContext(long traceId, long spanId, long parentId, byte flags) {
     this(traceId, spanId, parentId, flags, Collections.<String, String>emptyMap(), null);
   }
 
-  JaegerSpanContext(
+  protected JaegerSpanContext(
```

Comment:
who would use this ctor?

Reply:
It isn't exactly popular, but it is the more complete constructor. If the subclass wants to override the other constructor, it would need to use this constructor and pass in a custom factory for the last argument.

Reply:
Sorry, now that I see the constructor I understand what you mean. Will remove.